### PR TITLE
(feat) Added significant styling and card expansion logic for week view

### PR DESCRIPTION
**Pull request by rjatkinson2**

### CODE REVIEW

File: client/js/components/week/week-header.js [hunk 1]
```diff
@@ -4,8 +4,10 @@ var React = require('react');
 var WeekHeader = React.createClass({
   render: function () {
     return (
-      <div>
-        <h5>Week Header</h5>
+      <div className="date-bar">
+        <div className="arrow arrow-left"></div>
+        <h2>May 25 - May 30</h2>
```
Comment (benignbemine) [inline client/js/components/week/week-header.js]: just want to double check that these are supposed to be hardcoded for now

**Comment by kyleaowen** [comment]
looks good. we need to change events -> gigs. i can do that this evening

File: client/js/components/week/week-header.js (hunk 1 above)
Comment (rjatkinson2) [inline client/js/components/week/week-header.js:9]: correct sir. will fill dynamically once the api calls are running. looking forward to it!

**Comment by rjatkinson2** [comment]
Yea that's in addition to the card > bucket > board change too. Will look into that one tonight.

**Pull request merged**